### ISSUE TITLE
Using Nanopore and Illumina combined reads for Mitochondrial Genome Assembly 

### ISSUE THREAD

**Issue by manjumoorthy95** [ndierckx/NOVOPlasty#185]
Can we use both Nanopore and Illumina reads concatenated together into a single fastq file for the Mitochondrial genome assembly using Novoplasty? Or is Novoplasty specific for Illumina Data?

**Comment by ndierckx** [comment]
Hi,

Sorry forgot to reply! For now it works only on illumina data, although I have version for hybrid assembly (ONT&Illumina) that I will make available in the future when I have some time, but won't be in the next 2 months

**Comment by llk578496** [comment]
Hi @ndierckx,

I hope you’re doing well!

Could you please let me know if `nanopore` reads are now supported in `NOVOPlasty`?

Thank you very much!

Best,
Eddie

**Comment by ndierckx** [comment]
Hi,

Are you looking for a hybrid method (illumina + ONT) or only ONT?

**Comment by llk578496** [comment]
Hi @ndierckx ,

We are looking for the ONT only.

Best regards,
Eddie

**Comment by ndierckx** [comment]
Hi Sorry, was away for holiday.

I have new long read assembler, that works especially well on old ONT data (R9.4):
https://github.com/ndierckx/NOVOLoci/

I was planning to upload the code this month, if you want to try I can send you already

**Comment by llk578496** [comment]
Hi @ndierckx ,

Thank you very much! It would be wonderful if it could support the R10.4.1 reads.

Best regards,  
Eddie

**Comment by ndierckx** [comment]
Yeah of course it supports R10.4 too, but the difference compared to other assemblers is not as large as with R9.4
I will prepare the tool to upload

May I ask why you are looking for a new assembler, organelle assembly should be pretty straightforward with any kind of assembly tool, usually not that complex, especially with long reads..

**Comment by llk578496** [comment]
Hi @ndierckx,

I’m thrilled to hear that `NOVOLoci` supports the R10.4.1 reads. I apologize if I misunderstood your previous message.

Our team has primarily focused on viral and bacterial whole-genome sequencing (WGS) using ONT only. Despite some accuracy concerns, we have relied on ONT due to its cost-effectiveness, ease of use, speed, and our own limited funding.

Recently, we’ve been looking to expand our research capabilities to include fungal WGS. However, we quickly discovered that fungal genomes differ significantly from those of viruses and bacteria. Fungal genomes can include both chromosomes and mitochondria, and they may exhibit varying ploidy levels.

As we delved deeper into fungal genomics, we recognized the necessity of assembling mitochondrial DNA separately due to its distinct structural characteristics compared to chromosomal DNA.

In our search for a suitable assembler dedicated to mitochondrial genomes, we came across your remarkable tool, `NOVOPlasty`. However, we noticed that it currently supports only Illumina data. We also found that others have raised similar concerns, which prompted us to leave a comment here.

I apologize once again if my earlier misunderstanding stemmed from my limited knowledge of this area.

We are very interested in trying out your new tool, `NOVOLoci`. Please send the code to my email at lamkwongeddie.lee@connect.polyu.hk whenever it is convenient for you.

Thank you once again for your contributions to the field with both `NOVOPlasty` and `NOVOLoci`!

Best regards,  
Eddie

**Comment by ndierckx** [comment]
Hi,

Sorry for the delay, took a bit longer, but it is now online on github: https://github.com/ndierckx/NOVOLoci/blob/master/README.md

Let me know if you have any issues running it or installing it, I also added a circular option that can be used for organelle genomes
